gateway-js/src/index.ts

```diff
@@ -1271,3 +1271,4 @@ export {
 };
 
 export * from './datasources';
+export * from './schema-helper';
```

Comment:
Do we need to export these at the package level? I'd rather them not become part of the public API.